addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -141,24 +141,22 @@
         }
         &.s_image_gallery_indicators_arrows_boxed, &.s_image_gallery_indicators_arrows_rounded {
             .carousel {
-                .carousel-control-prev .fa,
-                .carousel-control-next .fa {
+                .carousel-control-prev .oi,
+                .carousel-control-next .oi {
                     text-shadow: none;
                 }
             }
         }
         &.s_image_gallery_indicators_arrows_boxed {
             .carousel {
-                .carousel-control-prev .fa:before {
-                    content: "\f104";
+                .carousel-control-prev .oi:before {
                     padding-right: 2px;
                 }
-                .carousel-control-next .fa:before {
-                    content: "\f105";
```

Comment:
Can you explain why you deleted it? `fa-angle-left` is used instead of `chevron` with arrows boxed.. Why not keep it ?

Reply:
Hello @bvr-odoo,

I thought we adapted `fa` everywhere so I thought there is no meaning to keeping fa-icons so I removed that as it cause unexpected behaviour ( refer to the attachment ). but now if we consider the already dropped snippet then I will have to restore that.
also, I didn't find an equivalent icon to the `fa-angle-right` in `oi` to replace it so I thought to keep the chevron for boxed too.

Before (with fa-angle-right) : 
![image](https://github.com/odoo/odoo/assets/125337508/512262e8-87ce-4db9-b73b-52590b841238)
After (with chevron): 
![image](https://github.com/odoo/odoo/assets/125337508/6dce65ab-fa34-4ab1-84e8-0def12a7d157)

Reply:
> I thought we adapted fa everywhere so I thought there is no meaning to keeping fa-icons so I removed that

Look at the commit https://github.com/odoo/odoo/commit/418413e4997a6b65eb7ad9e9ef8aba42805f1c0c that has adapted the icons, only the "fa-chevron" and "fa-arrow" have been adapted... 
It's not all the "fa" icons that have been changed as you say, so even if 
> it cause unexpected behaviour

:wink: I think we have to fix it and keep the fa-angle here

Reply:
@divy-odoo Please make the changes according to my last comment... Thanks :+1:

Reply:
@divy-odoo Please make the change according to [this comment](https://github.com/odoo/odoo/pull/122624/files#r1257793227) 
You must keep the same icon (fa-angle) for the "Boxed" arrows.